seeds/scripts/rerunners/wacca/_README.md

```diff
@@ -12,3 +12,10 @@ fs.writeFileSync('songlist.json', JSON.stringify(e));
 Then run the .js with node.
 
 Clearly not ideal, but it just works.
+
+If the above addition doesn't work, you can try adding the following instead (without the triple quotes):
+"""
+import fs from 'fs';
+fs.writeFileSync('songlist.json', JSON.stringify(a.concat(t)));
+"""
```

Comment:
Use [triple backticks](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code) for code blocks.

(also this isn't much different from the original dumping method, this time the songlist is just spread across 2 different arrays instead of one. and the variable names will continue to change because such is the nature of minified js code)